testsuite/integration-arquillian/test-apps/cors/angular-product/src/main/webapp/keycloak.json

```diff
@@ -1,8 +1,9 @@
 {
   "realm" : "cors",
   "realm-public-key" : "MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCrVrCuTtArbgaZzL1hvh0xtL5mc7o0NqPVnYXkLvgcwiC3BjLGw1tGEGoJaXDuSaRllobm53JBhjx33UNv+5z/UMG4kytBWxheNVKnL6GgqlNabMaFfPLPCF8kAgKnsi79NMo+n6KnSY8YeUmec/p2vjO2NjsSAVcWEQMVhJ31LwIDAQAB",
-  "auth-server-url" : "http://localhost-auth:8180/auth",
+  "auth-server-url" : "https://localhost-auth:8543/auth",
```

Comment:
Doesn't this effectively force https for adapters?

Reply:
Good eye! I believe I should use Maven filtering there and set `http`/`https` and port depending on SSL setting. Let me fix that.

Reply:
I believe this should be rather done by the deployment processor instead of Maven static build. See https://github.com/keycloak/keycloak/blob/master/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/DeploymentArchiveProcessor.java#L170-L183

Reply:
Unfortunately this is not an ordinary deployment, but rather a test app. It seems that our test apps are deployed as modules in WF and that means that we do not trigger `DeploymentArchiveProcessor`s on it. 

This one is heavily connected to @stianst's comment below. @stianst suggests that we should not disable trust manager and use proper trust store instead. So if we had a single point where we could inject a truststore into test applications, that would be lovely.

So I need an advice from you two (@hmlnarik and @stianst). Shall I try to refactor the existing applications to regular deployments? If so, it will take some time to do it (I bet the test suite will break up again). I see some benefits of doing this but having an opinion of someone more experienced might be helpful here.

Reply:
This app is deployed in `org.keycloak.testsuite.adapter.example.cors.CorsExampleAdapterTest.angularCorsProductExample()` Is `DeploymentArchiveProcessor` somehow not followed here?

Reply:
I think the issue is that the current `DeploymentArchiveProcessor` does not reflect angular path (i.e. no WEB-INF/ but webapp/ instead)

Reply:
I don't have much knowledge about the testing of adapters at all. This is probably a conversation for the team mailing list.